internal/route/repo/view.go

```diff
@@ -75,17 +75,19 @@ func renderDirectory(c *context.Context, treeLink string) {
 	if c.Data["HasDmpJson"].(bool) {
 		readDmpJson(c)
 	} else {
-		schemaUrl := getTemplateUrl() + "dmp/orgs"
-
-		var d dmpUtil
-		err := d.BidingDmpSchemaList(c, schemaUrl)
-		if err != nil && !c.IsInternalError() {
-			log.Warn("%v", err)
-			c.Flash.Warning(c.Tr("rcos.server.connect.failure"))
-		} else if err != nil && c.IsInternalError() {
-			log.Error(err.Error())
-			c.Error(fmt.Errorf(c.Tr("rcos.server.error")), "")
-			return
+		if c.Repo.BranchName == c.Repo.Repository.DefaultBranch {
```

Comment:
# Questions & Correction Requests
* If the default branch is changed from the repository settings screen, the Add DMP andf generate maDMPbutton will appear even if it is not master.

![image](https://github.com/NII-DG/gogs/assets/96706614/a2c23065-bac5-457d-98c6-d3dfca5d682f)